src/openforms/js/components/admin/form_design/registrations/zgw/ZGWOptionsFormFields.js

```diff
@@ -0,0 +1,308 @@
+import {produce} from 'immer';
+import PropTypes from 'prop-types';
+import React, {useContext} from 'react';
+import {useIntl} from 'react-intl';
+
+import {CustomFieldTemplate} from 'components/admin/RJSFWrapper';
+import {TextArea, TextInput} from 'components/admin/forms/Inputs';
+import Select from 'components/admin/forms/Select';
+import {ValidationErrorContext} from 'components/admin/forms/ValidationErrors';
+
+import {VariablePropertyModal} from './ZGWOptionsVariablesProperties';
+import {getChoicesFromSchema, getErrorMarkup, getFieldErrors} from './utils';
+
+const Wrapper = ({children}) => (
+  <form className="rjsf" name="form.registrationBackendOptions">
+    <CustomFieldTemplate displayLabel={false} errors={null}>
+      <fieldset id="root">{children}</fieldset>
+    </CustomFieldTemplate>
+  </form>
+);
+
+const ZGWFormFields = ({index, name, schema, formData, onChange}) => {
+  const intl = useIntl();
+  const validationErrors = useContext(ValidationErrorContext);
+
+  const {
+    zgwApiGroup = '',
+    zaakVertrouwelijkheidaanduiding = '',
+    zaaktype = '',
+    informatieobjecttype = '',
+    organisatieRsin = '',
+    medewerkerRoltype = '',
+    objecttype = '',
+    objecttypeVersion = '',
+    contentJson = '',
+  } = formData;
+
+  const onFieldChange = event => {
+    const {name, value} = event.target;
+    const updatedFormData = produce(formData, draft => {
+      draft[name] = value;
+    });
+    onChange(updatedFormData);
+  };
+
+  const buildErrorsComponent = field => {
+    const rawErrors = getFieldErrors(name, index, validationErrors, field);
+    return rawErrors ? getErrorMarkup(rawErrors) : null;
+  };
+
+  return (
+    <Wrapper>
+      <CustomFieldTemplate
```

Comment:
`CustomFieldTemplate` has a required `schema` prop type which is not passed. Looking at the rjsf implementation it does not actually lead to an exception but this could change at any moment

Reply:
Let's leave it at this - our dependencies are pinned and we are in no hurry to ugrade to a newer version. Most likely, we will even replace the dependency entirely with hand-rolled forms to provide a better user experience.

Let's add a comment warning/documenting this though - you can backlink to this comment with a short summary that we choose to ignore the `schema` prop as it doesn't seem to do anything.

Having some end-to-end tests to verify that the zgw forms can be configured would be good to have some protection against regressions.